app/models/docker/container.js

```diff
@@ -11,6 +11,9 @@ function ContainerViewModel(data) {
   this.Command = data.Command;
   this.Checked = false;
   this.Labels = data.Labels;
+  if (this.Labels && this.Labels['com.docker.compose.project']) {
+    this.StackName = this.Labels['com.docker.compose.project'];
```

Comment:
In swarm mode, containers (tasks) have the stack label too ("com.docker.stack.namespace")
Maybe we can grab it here to display linked stack for containers in swarm mode ?